<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" total="22" passed="22">
  <reporter-output>
  </reporter-output>
  <suite name="Surefire suite" duration-ms="199" started-at="2017-01-24T09:53:51Z" finished-at="2017-01-24T09:53:51Z">
    <groups>
    </groups>
    <test name="Surefire test" duration-ms="199" started-at="2017-01-24T09:53:51Z" finished-at="2017-01-24T09:53:51Z">
      <class name="com.jakub.ajamarks.repositories.StudentRepositoryTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.jakub.ajamarks.repositories.StudentRepositoryTest@35f983a6]" name="setUp" is-config="true" duration-ms="0" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="testFindStudentbyUserName()[pri:0, instance:com.jakub.ajamarks.repositories.StudentRepositoryTest@35f983a6]" name="testFindStudentbyUserName" duration-ms="0" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testFindStudentbyUserName -->
      </class> <!-- com.jakub.ajamarks.repositories.StudentRepositoryTest -->
      <class name="com.jakub.ajamarks.repositories.ClassroomRepositoryTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.jakub.ajamarks.repositories.ClassroomRepositoryTest@60f82f98]" name="setUp" is-config="true" duration-ms="4" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="testFindStudentsInClassroom()[pri:0, instance:com.jakub.ajamarks.repositories.ClassroomRepositoryTest@60f82f98]" name="testFindStudentsInClassroom" duration-ms="1" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testFindStudentsInClassroom -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.jakub.ajamarks.repositories.ClassroomRepositoryTest@60f82f98]" name="tearDown" is-config="true" duration-ms="0" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.jakub.ajamarks.repositories.ClassroomRepositoryTest -->
      <class name="com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest">
        <test-method status="PASS" signature="testSetUp()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="testSetUp" is-config="true" duration-ms="103" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSetUp -->
        <test-method status="PASS" signature="deleteTest()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="deleteTest" duration-ms="14" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteTest -->
        <test-method status="PASS" signature="testSetUp()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="testSetUp" is-config="true" duration-ms="0" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSetUp -->
        <test-method status="PASS" signature="delete_IllegalArgumentExceptionTest()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="delete_IllegalArgumentExceptionTest" duration-ms="2" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <exception class="java.lang.IllegalArgumentException">
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- delete_IllegalArgumentExceptionTest -->
        <test-method status="PASS" signature="getAllTest()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="getAllTest" duration-ms="2" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getAllTest -->
        <test-method status="PASS" signature="testSetUp()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="testSetUp" is-config="true" duration-ms="0" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSetUp -->
        <test-method status="PASS" signature="testSetUp()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="testSetUp" is-config="true" duration-ms="0" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSetUp -->
        <test-method status="PASS" signature="getClassroomStudentsByClassroomNameDescLastNameTest()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="getClassroomStudentsByClassroomNameDescLastNameTest" duration-ms="1" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getClassroomStudentsByClassroomNameDescLastNameTest -->
        <test-method status="PASS" signature="testSetUp()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="testSetUp" is-config="true" duration-ms="0" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSetUp -->
        <test-method status="PASS" signature="getClassroomStudentsByClassroomNameDescLastName_IllegalArgumentExceptionTest()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="getClassroomStudentsByClassroomNameDescLastName_IllegalArgumentExceptionTest" duration-ms="0" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <exception class="java.lang.IllegalArgumentException">
            <message>
              <![CDATA[classroom name can't be null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException: classroom name can't be null
	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:122)
	at com.jakub.ajamarks.services.showdataservices.StudentServiceImpl.getClassroomStudentsByClassroomNameDescLastName(StudentServiceImpl.java:76)
	at com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest.getClassroomStudentsByClassroomNameDescLastName_IllegalArgumentExceptionTest(StudentServiceImplTest.java:234)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:115)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:204)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:107)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:112)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getClassroomStudentsByClassroomNameDescLastName_IllegalArgumentExceptionTest -->
        <test-method status="PASS" signature="testSetUp()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="testSetUp" is-config="true" duration-ms="0" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSetUp -->
        <test-method status="PASS" signature="getClassroomStudentsByClassroomNameDescLastNumberTest()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="getClassroomStudentsByClassroomNameDescLastNumberTest" duration-ms="1" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getClassroomStudentsByClassroomNameDescLastNumberTest -->
        <test-method status="PASS" signature="getClassroomStudentsByClassroomNameDescLastNumber_IllegalArgumentExceptionTest()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="getClassroomStudentsByClassroomNameDescLastNumber_IllegalArgumentExceptionTest" duration-ms="0" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <exception class="java.lang.IllegalArgumentException">
            <message>
              <![CDATA[classroom number can't be less than 1]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException: classroom number can't be less than 1
	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:122)
	at com.jakub.ajamarks.services.showdataservices.StudentServiceImpl.getClassroomStudentsByClassroomNumberDescLastName(StudentServiceImpl.java:70)
	at com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest.getClassroomStudentsByClassroomNameDescLastNumber_IllegalArgumentExceptionTest(StudentServiceImplTest.java:211)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:115)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:204)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:107)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:112)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getClassroomStudentsByClassroomNameDescLastNumber_IllegalArgumentExceptionTest -->
        <test-method status="PASS" signature="testSetUp()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="testSetUp" is-config="true" duration-ms="0" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSetUp -->
        <test-method status="PASS" signature="getStudentByIdTest()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="getStudentByIdTest" duration-ms="1" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getStudentByIdTest -->
        <test-method status="PASS" signature="testSetUp()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="testSetUp" is-config="true" duration-ms="0" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSetUp -->
        <test-method status="PASS" signature="testSetUp()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="testSetUp" is-config="true" duration-ms="1" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSetUp -->
        <test-method status="PASS" signature="getStudentById_IllegalArgumentExceptionTest()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="getStudentById_IllegalArgumentExceptionTest" duration-ms="0" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <exception class="java.lang.IllegalArgumentException">
            <message>
              <![CDATA[id student can't be less than 1]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException: id student can't be less than 1
	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:122)
	at com.jakub.ajamarks.services.showdataservices.StudentServiceImpl.getStudentById(StudentServiceImpl.java:58)
	at com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest.getStudentById_IllegalArgumentExceptionTest(StudentServiceImplTest.java:167)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:115)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:204)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:107)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:112)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getStudentById_IllegalArgumentExceptionTest -->
        <test-method status="PASS" signature="testSetUp()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="testSetUp" is-config="true" duration-ms="0" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSetUp -->
        <test-method status="PASS" signature="getStudentByUserName_IllegalArgumentExceptionTest()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="getStudentByUserName_IllegalArgumentExceptionTest" duration-ms="1" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <exception class="java.lang.IllegalArgumentException">
            <message>
              <![CDATA[student's user name can't be null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException: student's user name can't be null
	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:122)
	at com.jakub.ajamarks.services.showdataservices.StudentServiceImpl.getStudentByUserName(StudentServiceImpl.java:64)
	at com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest.getStudentByUserName_IllegalArgumentExceptionTest(StudentServiceImplTest.java:189)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:115)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:204)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:107)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:112)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getStudentByUserName_IllegalArgumentExceptionTest -->
        <test-method status="PASS" signature="testSetUp()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="testSetUp" is-config="true" duration-ms="0" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSetUp -->
        <test-method status="PASS" signature="getStudentByUserNameTest()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="getStudentByUserNameTest" duration-ms="0" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getStudentByUserNameTest -->
        <test-method status="PASS" signature="getStudentsWithGivenMarkTest()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="getStudentsWithGivenMarkTest" duration-ms="39" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getStudentsWithGivenMarkTest -->
        <test-method status="PASS" signature="testSetUp()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="testSetUp" is-config="true" duration-ms="0" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSetUp -->
        <test-method status="PASS" signature="testSetUp()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="testSetUp" is-config="true" duration-ms="1" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSetUp -->
        <test-method status="PASS" signature="getStudentsWithGivenMark_IllegalArgumentExceptionTest()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="getStudentsWithGivenMark_IllegalArgumentExceptionTest" duration-ms="0" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <exception class="java.lang.IllegalArgumentException">
            <message>
              <![CDATA[mark mast be from 1 to 6 ]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException: mark mast be from 1 to 6 
	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:122)
	at com.jakub.ajamarks.services.showdataservices.StudentServiceImpl.getStudentsWithGivenMark(StudentServiceImpl.java:82)
	at com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest.getStudentsWithGivenMark_IllegalArgumentExceptionTest(StudentServiceImplTest.java:262)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:115)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:204)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:107)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:112)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getStudentsWithGivenMark_IllegalArgumentExceptionTest -->
        <test-method status="PASS" signature="getStudentsWithoutGivenMarkTest()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="getStudentsWithoutGivenMarkTest" duration-ms="1" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getStudentsWithoutGivenMarkTest -->
        <test-method status="PASS" signature="testSetUp()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="testSetUp" is-config="true" duration-ms="0" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSetUp -->
        <test-method status="PASS" signature="testSetUp()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="testSetUp" is-config="true" duration-ms="1" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSetUp -->
        <test-method status="PASS" signature="testSetUp()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="testSetUp" is-config="true" duration-ms="1" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSetUp -->
        <test-method status="PASS" signature="getStudentsWithoutGivenMark_IllegalArgumentExceptionTest()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="getStudentsWithoutGivenMark_IllegalArgumentExceptionTest" duration-ms="0" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <exception class="java.lang.IllegalArgumentException">
            <message>
              <![CDATA[mark mast be from 1 to 6 ]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException: mark mast be from 1 to 6 
	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:122)
	at com.jakub.ajamarks.services.showdataservices.StudentServiceImpl.getStudentsWithoutGivenMark(StudentServiceImpl.java:100)
	at com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest.getStudentsWithoutGivenMark_IllegalArgumentExceptionTest(StudentServiceImplTest.java:291)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:115)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:204)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:107)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:112)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getStudentsWithoutGivenMark_IllegalArgumentExceptionTest -->
        <test-method status="PASS" signature="testSetUp()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="testSetUp" is-config="true" duration-ms="1" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSetUp -->
        <test-method status="PASS" signature="saveStudentTest()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="saveStudentTest" duration-ms="0" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- saveStudentTest -->
        <test-method status="PASS" signature="saveStudent_IllegalArgumentExceptionTest()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="saveStudent_IllegalArgumentExceptionTest" duration-ms="0" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <exception class="java.lang.IllegalArgumentException">
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- saveStudent_IllegalArgumentExceptionTest -->
        <test-method status="PASS" signature="updateByUserNameTest()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="updateByUserNameTest" duration-ms="1" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updateByUserNameTest -->
        <test-method status="PASS" signature="testSetUp()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="testSetUp" is-config="true" duration-ms="0" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSetUp -->
        <test-method status="PASS" signature="testSetUp()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="testSetUp" is-config="true" duration-ms="0" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSetUp -->
        <test-method status="PASS" signature="updateByUserName_IllegalArgumentExceptionTest()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="updateByUserName_IllegalArgumentExceptionTest" duration-ms="0" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <exception class="java.lang.IllegalArgumentException">
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updateByUserName_IllegalArgumentExceptionTest -->
        <test-method status="PASS" signature="testSetUp()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="testSetUp" is-config="true" duration-ms="1" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSetUp -->
        <test-method status="PASS" signature="updateByUserName_NullPointerExceptionTest()[pri:0, instance:com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest@7f690630]" name="updateByUserName_NullPointerExceptionTest" duration-ms="0" started-at="2017-01-24T10:53:51Z" finished-at="2017-01-24T10:53:51Z">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[student name can't be null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: student name can't be null
	at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:228)
	at com.jakub.ajamarks.services.showdataservices.StudentServiceImpl.updateByUserName(StudentServiceImpl.java:39)
	at com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest.updateByUserName_NullPointerExceptionTest(StudentServiceImplTest.java:113)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:115)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:204)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:107)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:112)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updateByUserName_NullPointerExceptionTest -->
      </class> <!-- com.jakub.ajamarks.services.showdataservices.StudentServiceImplTest -->
    </test> <!-- Surefire test -->
  </suite> <!-- Surefire suite -->
</testng-results>
